### ISSUE TITLE
Pagination: invalid aria-current value on pagination__item

### ISSUE THREAD

**Issue by yzhao888** [eBay/ebayui-core#302]
# Bug Report

## Description

in ebay-pagination generated pagination__item that are not currently selected, they have aria-current='', this is an invalid value.
per https://tink.uk/using-the-aria-current-attribute/, for page not currently selected, they should not have aria-current attribute at all.
This is also reported in WAE tool. 
 
## Workaround

No
## Screenshots

N/A

**Comment by ianmcburnie** [comment]
Super quick fix might be to set the value to `false` instead of empty, but yes ultimately the attribute should be removed if the item is not the current item.
